charts/rundeck/scripts/bootstrap.sh

```diff
@@ -17,8 +17,10 @@ if test -f "$FRAMEWORK_SRC"; then
     cp "$FRAMEWORK_SRC" /tmp/remco-partials/framework/framework-custom.properties
 fi
 
-echo "Copying custom plugins"
-cp -r /mnt/plugins/. /home/rundeck/libext
+if test -d /mnt/plugins; then
+    echo "Copying custom plugins"
+    cp -r /mnt/plugins/. /home/rundeck/libext
+fi
```

Comment:
maybe an else case with 'Skipping custom plugin deployment since it seems not to be mounted'.

So if that was accidental, there is at least a hint. What do you think?

Reply:
added else + echo